packages/admin-next/dashboard/src/v2-routes/taxes/tax-region-list/components/region-list-table/tax-region-list-table.tsx

```diff
@@ -45,23 +50,22 @@ export const TaxRegionListTable = () => {
     <Container className="divide-y p-0">
       <div className="flex items-center justify-between px-6 py-4">
         <Heading level="h2">{t("taxes.domain")}</Heading>
+
         <Link to="/settings/taxes/create">
           <Button size="small" variant="secondary">
             {t("actions.create")}
           </Button>
         </Link>
       </div>
+
```

Comment:
use `asChild` so the button styles are used for the focus state rather than the Link built in style.

```suggestion
          <Button size="small" variant="secondary" asChild>
              <Link to="/settings/taxes/create">
                  {t("actions.create")}
              </Link>
          </Button>
      </div>
```